README.md

```diff
@@ -26,31 +26,51 @@ Next follow these instructions.
     PASSWORD: osp
     HOST: localhost
     ``` 
-    You may run the following commands for local setup of DB:
+    You may run the following commands for local setup of DB in Linux:
 
     ```
+    cd open-source-programs-backend
     sudo -i -u postgres
     createuser osp --pwprompt
     psql
     CREATE DATABASE osp;
     \c osp;
     GRANT ALL PRIVILEGES ON DATABASE osp to osp;
     ```
+2. you need to download API key file from your user-settings on Zulip. The file you download is named as 'download' or rename that to 'download'.
+3. Put that download file in the project's directory.
 
-
-2. To start the server:
+4. Move into the project's directory.
 
     ```
     cd open-source-programs-backend
+    ````
+5. You don't need to create virtual environment while working in Linux.
+    `````
     virtualenv venv
     source venv/bin/activate
     pip install -r requirements.txt
```

Comment:
When I did this, it worked only then i wrote ```python3 -m pip install requirements.txt```.

Also, prior to this, I had to install the dependencies for psycopg2, mainly python3-dev and libpq-dev. The the installation worked for me.

Reply:
If you're using python 3.8 you can install the requirements just by writing 
``` 
pip3 install - r requirements.txt
```
@Soumi7 no need to install anything else

Reply:
@Aaishpra I see. I was using python 3.6 hence the issues. Okay.